src/layouts/Settings/Settings.tsx

```diff
@@ -34,11 +36,13 @@ import { SocialMediaSettings } from "./SocialMediaSettings"
 
 export const Settings = (): JSX.Element => {
   const { siteName } = useParams<{ siteName: string }>()
+  const { userId } = useLoginContext()
+  const isGithubUser = userId !== "Unknown user" && !!userId
```

Comment:
could we add a note explaining hwy this means that it is a github user?

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/1316/commits/4c6368275c31a6c5cf27bacac104a1ac9207f9bb